Description: disable mongodb_useauth for proper pkg installation
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 graylog2-server (0.9.6-1) unstable; urgency=low
 .
   * Initial release.
Author: Andrey Domas <andrey.domas@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- graylog2-server-0.9.6.orig/misc/graylog2.conf
+++ graylog2-server-0.9.6/misc/graylog2.conf
@@ -10,7 +10,7 @@ elasticsearch_index_name = graylog2
 force_syslog_rdns = false
 
 # MongoDB Configuration
-mongodb_useauth = true
+mongodb_useauth = false
 mongodb_user = grayloguser
 mongodb_password = 123
 mongodb_host = localhost
